src/inmanta/protocol/methods.py

```diff
@@ -815,7 +815,15 @@ def set_parameters(tid: uuid.UUID, parameters: list):
 # Get parameters from the agent
 
 
-@method(path="/agent_parameter", operation="POST", server_agent=True, timeout=5, arg_options=AGENT_ENV_OPTS, client_types=[])
+@method(
+    path="/agent_parameter",
+    operation="POST",
+    server_agent=True,
+    timeout=5,
+    arg_options=AGENT_ENV_OPTS,
+    client_types=[],
+    reply=False,
```

Comment:
I'm quite confused by this. The method name and docstring heavily imply that this returns relevant information, and its handler does in fact return something. Yet by my understanding of `reply=False` we will now no longer return anything. What am I missing?

Reply:
I see now that it returns nothing more than a status code. I think we should update the docstring, because it presents a false picture.

Reply:
Iirc, the `reply` option only has a meaning in the context of our client. If set to True, our client will not wait for a reply from the server. Which does not imply that the server doesn't return anything.

Reply:
If I understand correctly it's used by the server as well. Not that the handler can not return a reply, but that it [is not expected to](https://github.com/inmanta/inmanta-core/blob/25e7811826d2c9e1e1c1f9f51779b28acacb1b3c/src/inmanta/protocol/rest/__init__.py#L475C37-L475C37).

Reply:
Indeed. I didn't know that.